lib/iso_builder.py

```diff
@@ -46,9 +46,11 @@ def _run_mock_command(self, cmd):
             self.common_config.get('distro_name'),
             self.common_config.get('distro_version'),
             self.common_config.get('arch_and_endianness'))
+        mock_config_file = self.config.get('mock_config').get(distro.lsb_name).get(
+            distro.version)
```

Comment:
This should probably be in the previous patch "Make ISO creation get distro version specific Mock config file "